bitnami/airflow/templates/deployment-scheduler.yaml

```diff
@@ -128,6 +128,9 @@ spec:
                 {{- if .Values.postgresql.enabled }}
                   name: {{ template "airflow.postgresql.fullname" . }}
                   key: postgresql-password
+                {{- else if .Values.externalDatabase.existingSecret}}
+                  name: {{ .Values.externalDatabase.existingSecret }}
+                  key: db-password
```

Comment:
I suggest adding a helper function like the one below:

```golang
{{/*
Get the postgresql credentials secret.
*/}}
{{- define "airflow.postgresql.secretName" -}}
{{- if .Values.externalDatabase.existingSecret -}}
    {{- printf "%s" .Values.externalDatabase.existingSecret -}}
{{- else -}}
    {{- printf "%s" (include "airflow.postgresql.fullname" .) -}}
{{- end -}}
{{- end -}}
```

and then replacing 

```yaml
                {{- if .Values.postgresql.enabled }}
                  name: {{ template "airflow.postgresql.fullname" . }}
                  key: postgresql-password
                {{- else if .Values.externalDatabase.existingSecret}}
                  name: {{ .Values.externalDatabase.existingSecret }}
                  key: db-password
                {{- else }}
```

with

```
                {{- if .Values.postgresql.enabled }}
                  name: {{ template "airflow.postgresql.secretName" . }}
                  key: postgresql-password
                {{- else }}
```

The same should also be done for the Redis bits

Reply:
I'm now determining the `secretKeyRef.name` entirely in the template:

https://github.com/bitnami/charts/pull/1917/commits/2ca88f1b472b796e506d9bef21486cb6aaa79b3f#diff-404f4950f79b92ad20c70deadbe1bb17